src/components/Auth/Auth.jsx

```diff
@@ -40,21 +120,35 @@ function Auth({ isRegistration, setIsAuth }) {
                     name="name"
                     id="formName"
                     placeholder="Имя"
+                    value={formData.name}
+                    onChange={handleChange}
+                    $error={errors.name}
                   />
                 )}
                 <InputAuth
                   type="text"
                   name="login"
                   id="formLogin"
                   placeholder="Эл. почта"
+                  value={formData.login}
+                  onChange={handleChange}
+                  $error={errors.login}
                 />
                 <InputAuth
                   type="password"
                   name="password"
                   id="formPassword"
                   placeholder="Пароль"
+                  value={formData.password}
+                  onChange={handleChange}
+                  $error={errors.password}
                 />
-                <ButtonEnter id="btnEnter" onClick={handleLogin}>
+                <p style={{ color: "red" }}>{textError}</p>
+                <ButtonEnter
+                  id="btnEnter"
+                  onClick={handleLogin}
+                  disabled={textError ? true : false}
```

Comment:
тоже может быть сокращено до такой конструкции

```suggestion
                  disabled={!!textError}
```